src/ioctls/vcpu.rs

```diff
@@ -222,7 +222,7 @@ impl VcpuFd {
     /// let vcpu = vm.create_vcpu(0).unwrap();
     /// let regs = vcpu.get_regs().unwrap();
     /// ```
-    #[cfg(not(any(target_arch = "arm", target_arch = "aarch64")))]
+    #[cfg(not(any(target_arch = "arm", target_arch = "aarch64", target_arch = "riscv64")))]
     pub fn get_regs(&self) -> Result<kvm_regs> {
```

Comment:
At this point I wonder if it should just be `#[cfg(target_arch = "x86_64")]`

Reply:
Good point, I'll check if there is anything similar to this

Reply:
I think it's best to confine the checks only to the architectures we support with this crate.

Reply:
FWIW - your solution is fine as is.

Reply:
> I think it's best to confine the checks only to the architectures we support with this crate.

I will purpose that in another PR :) to discuss whether we should drop `target_arch` predicates to architectures other than `x86`, `arm` and `riscv`